volttron/platform/certs.py

```diff
@@ -155,7 +157,7 @@ def _mk_cacert(**kwargs):
         # set CA to true
         x509.BasicConstraints(ca=True, path_length=1),
         critical=True
-    ).serial_number(1).add_extension(
+    ).serial_number(int(time.time())).add_extension(
```

Comment:
Should we base it on localtime or utctime...in my implementation I had it using utctime..just because we do that everywhere else!

Reply:
Not needed in this context